python-sdk/tutorials/deploy-with-prebuilt-docker-images/pytorch-prebuilt-docker-image.ipynb

```diff
@@ -0,0 +1,938 @@
+        },
+        "papermill": {
+          "duration": 0.2322,
+          "end_time": "2021-05-18T20:36:45.333348",
+          "exception": false,
+          "start_time": "2021-05-18T20:36:45.101148",
+          "status": "completed"
+        },
+        "tags": []
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Prerequisites\n",
+        "\n",
+        "If you are using an Azure Machine Learning Notebook VM, you are all set. Otherwise, go through the [Configuration](../../../../configuration.ipynb) notebook to install the Azure Machine Learning Python SDK and create an Azure ML `Workspace`"
+      ],
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        },
+        "papermill": {
+          "duration": 0.044699,
+          "end_time": "2021-05-18T20:36:45.433973",
+          "exception": false,
+          "start_time": "2021-05-18T20:36:45.389274",
+          "status": "completed"
+        },
+        "tags": []
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "# Check core SDK version number\n",
+        "import azureml.core\n",
+        "\n",
+        "print(\"SDK version:\", azureml.core.VERSION)"
+      ],
+      "outputs": [],
+      "execution_count": null,
+      "metadata": {
+        "collapsed": true,
+        "gather": {
+          "logged": 1620920632022
+        },
+        "jupyter": {
+          "outputs_hidden": false,
+          "source_hidden": false
+        },
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        },
+        "papermill": {
+          "duration": 0.068634,
+          "end_time": "2021-05-18T20:36:45.559675",
+          "exception": false,
+          "start_time": "2021-05-18T20:36:45.491041",
+          "status": "completed"
+        },
+        "tags": []
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Initialize workspace\n",
+        "Initialize a [Workspace](https://docs.microsoft.com/azure/machine-learning/service/concept-azure-machine-learning-architecture#workspace) object from the existing workspace you created in the Prerequisites step. `Workspace.from_config()` creates a workspace object from the details stored in `config.json`."
+      ],
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        },
+        "papermill": {
+          "duration": 0.044633,
+          "end_time": "2021-05-18T20:36:45.655492",
+          "exception": false,
+          "start_time": "2021-05-18T20:36:45.610859",
+          "status": "completed"
+        },
+        "tags": []
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "from azureml.core.workspace import Workspace\n",
+        "\n",
+        "ws = Workspace.from_config()\n",
+        "print(\n",
+        "    \"Workspace name: \" + ws.name,\n",
```

Comment:
just print out the object, remove the big print statement

Reply:
done